### ISSUE TITLE
IMB Activiteiten en tijd

### ISSUE THREAD

**Issue by SCvanVeen** [EdwinSmink/Gemboxx#32]
Bij user story 9 van IMB is het de bedoeling dat mensen geattendeerd kunnen worden als er al een maand niks is gebeurd met een bepaald project. Is het mogelijk om dit met de huidige functionaliteit te doen?

Ik zie wel een aanvraagdatum in de proces-widget, maar ik kan deze aanvraagdatum niet terugvinden in de case zelf.

**Comment by EdwinSmink** [comment]
Ja, dat kun je door een deadline te zetten op een taak. Welke taak dat zou moeten zijn weet ik niet maar we kunnen daar wel een truuk voor verzinnen. On deadline passed kun je dan de activiteit 'stuur herinneringsmail' triggeren.
